community/modules/scripts/ramble-setup/main.tf

```diff
@@ -45,16 +45,31 @@ locals {
   )
 
   deps_content = templatefile(
-    "${path.module}/templates/install_ramble_deps.yml.tpl",
+    "${path.module}/templates/install_ramble_deps.yml.tftpl",
     {
-      ramble_ref = var.ramble_ref
+      virtualenv_path = var.ramble_virtualenv_path
     }
   )
 
   install_ramble_deps_runner = {
     "type"        = "ansible-local"
     "content"     = local.deps_content
     "destination" = "install_ramble_deps.yml"
+    "args"        = "-e virtualenv_path=${var.ramble_virtualenv_path}"
```

Comment:
There is some duplication going on here between the args and the templating. 

In this case `virtualenv_path`is getting written directly into the file via the templating that happens on L47. Then you are also passing the same variable in via ansible args. I would suggest only doing one or the other.

There is a general preference to have scripts instead of templates when possible. My suggestion would be to change install_ramble_deps back to a script and then just pass the path as an arg.

Reply:
Thanks! I honestly wasn't sure what the `args` part was for here.